blog/2023-04-24-no-sql-database-comparison/index.mdx

```diff
@@ -0,0 +1,347 @@
+complementary services that scale independently of each other. That makes it
+easier to adjust capacity either up or down as demand changes.
+
+## Data hosting with customer
+
+Where you host a database can decide how much control you have over it. A purely
+cloud-based service might make it harder to move your data should your needs
+change. Even if you start out with a cloud-based service, the option to
+self-host gives you flexibility and avoids lock-in.
+
+## Comparing Tigris with MongoDB and DynamoDB {#mongodb-vs-dynamodb-comparison}
+
+When making a choice between Tigris, MongoDB, and DynamoDB you need to take each
+of the above criteria into account. Here's how each of them compares.
+
+<table>
+  <tr>
+    <td></td>
+    <td>Tigris</td>
+    <td>DynamoDB</td>
+    <td>MongoDB</td>
+  </tr>
+  <tr>
+    <td>
+      <strong>Interactive ACID Transactions</strong>
+    </td>
+    <td>
+      <a href="https://www.tigrisdata.com/docs/concepts/database/transaction/">
+        yes
+      </a>
+    </td>
+    <td>no</td>
+    <td>
+      yes (
+      <a href="https://www.mongodb.com/docs/manual/reference/read-concern-linearizable/#performance-comparisons">
+        blocking reads
+      </a>
+      ,<a href="https://www.mongodb.com/docs/manual/core/transactions-sharded-clusters/#performance">
+        {" "}
+        significant performance cost
+      </a> for cross-shard transactions, read/write concern)
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <strong>Strongly Consistent Global Secondary Indexes</strong>
+    </td>
+    <td>yes</td>
+    <td>no</td>
+    <td>
+      <a href="https://www.mongodb.com/community/forums/t/indexes-on-sharded-collection/152795/2">
+        no
+      </a>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <strong>Database Branching</strong>
+    </td>
+    <td>
+      <a href="https://www.tigrisdata.com/docs/concepts/database/branching/">
+        yes
+      </a>
+    </td>
+    <td>no</td>
+    <td>no</td>
+  </tr>
+  <tr>
+    <td>
+      <strong>Integrated Full Text Search</strong>
+    </td>
+    <td>
+      <a href="https://www.tigrisdata.com/docs/concepts/database/search/">
+        yes
+      </a>
+    </td>
+    <td>no</td>
+    <td>yes</td>
+  </tr>
+  <tr>
+    <td>
+      <strong>Automatic Database Sharding</strong>
+    </td>
+    <td>yes</td>
+    <td>yes</td>
+    <td>no</td>
+  </tr>
+  <tr>
+    <td>
+      <strong>Cloud-native Architecture</strong>
+    </td>
+    <td>yes</td>
+    <td>yes</td>
+    <td>no</td>
+  </tr>
+  <tr>
+    <td>
+      <strong>Data Hosting with Customer</strong>
+    </td>
+    <td>yes</td>
+    <td>yes</td>
+    <td>no</td>
+  </tr>
+</table>
+
+## Tigris: the MongoDB and DynamoDB alternative
+
+We built Tigris to give developers the consistency of ACID transactions with the
+flexibility of NoSQL at a fraction of the cost of alternatives. Compared to
+MongoDB and DynamoDB, Tigris gives you:
+
+- **Consistent data for mission-critical applications:** With Tigris, you can be
```

Comment:
Are there more docs we can link to here? e.g. we have DB branching and MongoDB compatibility